m/configDTOutputNetCDFL1.m

```diff
@@ -445,6 +445,14 @@
     '_FillValue'    default_fill_value
     'sources'       []};
 
+  var_attr_list.bb700 = { % -FC
+    'long_name'     'backscatter 700nm'
+    'standard_name' 'backscatter_700'
+    'units'         '1'
+    'coordinates'   'time depth latitude longitude'
+    '_FillValue'    default_fill_value
+    'sources'       []};
+  
```

Comment:
Although this is just the default configuration, and the user can add or remove or rename the variables
according to their needs, for coherence with the already defined variables I would rename this to
"backscatter_700" with attributes:
  -  long_name:  700 nm wavelength backscatter
  -  standard_name: 700nm_backscatter